modules/kit-elements-defaults/assets/js/editor/hooks/data/fill-defaults-on-drop.js

```diff
@@ -19,16 +20,26 @@ export default class FillDefaultsOnDrop extends $e.modules.hookData.Dependency {
 
 		const elementSettings = getElementDefaults( widgetType || elType );
 
-		if ( ! elementSettings ) {
+		if ( ! isPopulatedObject( elementSettings ) ) {
 			return true;
 		}
 
+		const settings = { ...( args.model.settings || {} ), ...elementSettings };
```

Comment:
```suggestion
		const settings = { 
			...( args.model.settings || {} ),
			...elementSettings,
		};
```

I know it doesn't matter, but I think it puts less strain on the eyes when reading it, WDYT?

Reply:
Done